composer.json

```diff
@@ -10,7 +10,8 @@
     ],
     "require": {
         "php": "^8.1",
-        "illuminate/database": "^10.0"
+        "illuminate/database": "^10.0",
+        "singlestoredb/singlestoredb-laravel": "^1.5.1"
```

Comment:
Please move the package to the `require-dev` section. It shouldn't be included but installed separately.